include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -322,6 +325,39 @@ namespace parallel
                            const std::vector<dealii::CellData<dim>> &cells,
                            const SubCellData &subcelldata) override;
 
+      /**
+       * Implementation of the same function as in the base class.
+       *
+       * @param other_tria The triangulation to be copied. It can be a serial
+       *        Triangulation or a parallel::distributed::Triangulation. Both
+       *        can have been refined already.
+       *
+       * @note This function uses the partitioner registered with
+       *       register_partitioner().
+       */
+      void
+      copy_triangulation(
+        const dealii::Triangulation<dim, spacedim> &other_tria) override;
+
+      /**
+       * Register a partitioner, which is used within the method
+       * copy_triangulation.
+       *
+       * @param partitioner A partitioning function, which takes as input argument
+       *                    a reference to the triangullation to be partitioned
+       *                    and the number of partitions to be created.
```

Comment:
Can you say what the function is supposed to do? I suspect that it is supposed to set subdomain ids? And that the triangulation it gets is a copy of a triangulation that can be changed freely. Maybe like this:
```suggestion
       *                    and the number of partitions to be created. The function needs to set subdomain
       *                    ids for each active cell of the given triangulation, with values between zero (inclusive)
       *                    and the second argument to the function (exclusive).
```